src/peergos/shared/util/Futures.java

```diff
@@ -9,6 +9,7 @@
 
 public class Futures {
 
+    @JsMethod
     public static final <T> CompletableFuture<T> of(T val) {
```

Comment:
I can't find where this is used in JS?

Reply:
The calling code in JS was removed during a refactor. The .of() method is really useful though. I think it should be accesisble to Javascript. It would tidy up some code in gwt.js for a start.

Reply:
gwt.js is a precursor to peergoslib so it can't depend on it.

Reply:
Every JS annotation makes it harder to refactor safely, so I generally try to minimise them.